packages/millicast-sdk/src/View.js

```diff
@@ -243,27 +247,34 @@ export default class View extends BaseWebRTC {
 
     const workerBlob = new Blob([workerString])
     const workerURL = URL.createObjectURL(workerBlob)
+    this.worker = new Worker(workerURL)
 
     webRTCPeerInstance.on('track', (trackEvent) => {
-      if (trackEvent.track?.kind !== 'video') return
-      const worker = new Worker(workerURL)
+      this.trackMediaId[trackEvent.transceiver?.mid] = trackEvent.track
```

Comment:
Looks like this is also a map between track's media id and the track itself, if yes, we should name it `trackMediaIdMap`, and also add comment to explain the purpose of this object.

Reply:
As it will be used as an object, I think it is good not to include map in it, will change it to `tracksMidValues` for better clarity